packages/components/Checkbox/SPEC.md

```diff
@@ -122,13 +122,16 @@ The `Checkbox` control supports the `unchecked` and `checked` appearances. It do
 
 The `Checkbox` control supports a rounded `square` (default) and `circular` shape variants.
 
-Note: Not Supported on Android
+Note:
+
+- Circular Checkbox is not supported on Android
+- Square Checkbox is not supported on iOS
```

Comment:
A markdown table here might be nice

|Platform|Square|Circular
|---|---|---|
|Windows|✅|✅|
|iOS|❌|✅|
|macOS|✅|❌|
|Android|✅|❌|

Reply:
Changes completed and approved , hence resolving.